proto/api/v1/summary_service.proto

```diff
@@ -17,6 +17,7 @@ message SummaryCountsResponse {
   int64 num_nodes = 6;
 }
 
+// TODO(ROX-24528): This API is deprecated, remove it in 4.7
 service SummaryService {
   rpc GetSummaryCounts(Empty) returns (SummaryCountsResponse) {
```

Comment:
I suggest to add a warning to the log in `GetSummaryCounts()` that the API is deprecated as well.